app/frontend/src/internationalization/trans/chefs/pu/pu.json

```diff
@@ -0,0 +1,898 @@
+    "confirmDeletionMsg": "کی تسیں یکینی تور 'تے اس API کنجی نوں مٹاؤنا چاہندے ہو؟",
+    "delete": "مٹاؤ",
+    "confirmRestore": "ریسٹور دی پشٹی کرو"
+  },
+  "administerUser": {
+    "userDetails": "اپبھوگتا ویروے",
+    "openSSOConsole": "SSO کنسول کھولھو"
+  },
+  "adminPage": {
+    "forms": "فارم",
+    "users": "اپبھوگتا",
+    "developer": "وکاسکار",
+    "infoLinks": "جانکاری لنک",
+    "metrics": "میٹرکس"
+  },
+  "adminUsersTable": {
+    "search": "کھوج",
+    "loadingText": "لوڈ ہو رہا ہے... کرپا کرکے اڈیک کرو",
+    "admin": "ایڈمن",
+    "fullName": "پورا نانم",
+    "userID": "یوجر آئیڈی",
+    "created": "بنایا",
+    "actions": "کاروائیاں"
+  },
+  "adminVersions": {
+    "exportDesign": "نریات ڈزائین",
+    "versions": "سنسکرن",
+    "status": "ستھتی",
+    "created": "بنایا",
+    "lastUpdated": "آخری وار اپڈیٹ کیتا",
+    "actions": "کاروائیاں",
+    "published": "پرکاشت",
+    "unpublished": "اپرکاشت",
+    "version": "سنسکرن {versionNo}",
+    "notificationMsg": "فارم ڈزائین نوں لوڈ کرن دوران اکّ ترٹی اتپنّ ہوئی۔"
+  },
+  "developer": {
+    "user": "اپبھوگتا",
+    "name": "نام",
+    "userName": "اپبھوگتا نام",
+    "JWTContents": "JWT سمگری",
+    "JWTContentsSBTxt": "JWT سمگری کلپبورڈ 'تے کاپی کیتی گئی",
+    "JWTContentsTTTxt": "JWT سمگری نوں کلپبورڈ وچّ کاپی کرو",
+    "JWTToken": "JWT ٹوکن",
+    "JWTTokenSBTxt": "JWT ٹوکن کلپبورڈ 'تے کاپی کیتا گیا",
+    "JWTTokenTTTxt": "JWT ٹوکن نوں کلپبورڈ وچّ کاپی کرو",
+    "chefsAPI": "CHEFS API",
+    "RBACSBTxt": "RBAC جواب کلپبورڈ 'تے کاپی کیتا گیا",
+    "RBACTTTxt": "RBAC جواب نوں کلپبورڈ 'تے کاپی کرو",
+    "notificationMsg": "RBAC توں اپبھوگتا پراپت کرن وچّ اسپھل, کنسول دیکھو",
+    "notificationConsErr": "RBAC توں اپبھوگتا پراپت کرن وچّ ترٹی"
+  },
+  "fcProactiveHelp": {},
+  "baseAuthButton": {
+    "logout": "لاگّ آؤٹ, باہر آؤنا",
+    "login": "لاگن"
+  },
+  "baseDialog": {
+    "defaultText": "ڈپھالٹ ٹیکسٹ",
+    "ok": "ٹھیک ہے",
+    "continue": "جاری رکھو",
+    "cancel": "ردّ کرو",
+    "custom": "پرتھا"
+  },
+  "baseSecure": {
+    "about": "بارے",
+    "loginInfo": "تہانوں اس وشیشتا دی ورتوں کرن لئی لوگئن کرنا چاہیدا ہے۔",
+    "login": "لاگن",
+    "401NotAuthorized": "401: ادھکارت نہیں۔ :",
+    "401ErrorMsg": "تہاڈا خاتا سہیہ ڈھنگ نال سیٹئپّ نہیں کیتا گیا ہے۔<�br /> کرپا کرکے سمپرک کرو",
+    "403Forbidden": "403: ورجت۔ :",
+    "403ErrorMsg": "اس پنے نوں {idp} پرمانیکرن دی لوڑ ہے۔",
+    "401UnAuthorized": "401: انئدھکارت۔ :",
+    "401UnAuthorizedErrMsg": "تہانوں اس پنے تکّ پہنچ کرن دی اجازت نہیں ہے۔"
+  },
+  "formDesigner": {
+    "formDesign": "فارم ڈزائین",
+    "exportDesign": "نریات ڈزائین",
+    "importDesign": "ڈزائین آیات کرو",
+    "important": "مہتوپورن",
+    "formDesignInfoA": "جدوں تسیں اس فارم نوں بناؤنا پورا کر لیندے ہو تاں <�strong>سیو ڈزائین<�/strong> بٹن دی ورتوں کرو۔",
+    "formDesignInfoB": "تہاڈے اپبھوگتا نوں اہ فارم جمھاں کراؤن لئی <�strong>SUBMIT بٹن دتا گیا ہے اتے اسنوں سرکھئت کرن توں بائد کریاشیل ہو جاویگا۔",
```

Comment:
It looks like there is some kind of special character here, and it's not formatting as STRONG:
![image](https://github.com/bcgov/common-hosted-form-service/assets/35532993/115d6b74-5898-441f-977d-7b3fd164a171)